### DISCUSSION TITLE
Add Dry/Wet Settings

### DISCUSSION THREAD

**Discussion by chrsrns** [wwmm/easyeffects#1587] [Ideas]
I think it is beneficial for some cases, like in noise reduction, to be able to "mix" the input audio with the output audio. This way, in the case of noise reduction, there is some control on how much noise is removed from the audio.

**Comment by wwmm** [comment]
I could try to add this control to the noise reduction plugin. When possible open an issue about this to help me to remember to do it.

**Reply by chrsrns** [reply]
Will do! Thank you for considering!
